docs/features/refout.md

```diff
@@ -67,10 +68,10 @@ As mentioned above, there may be further refinements after C# 7.1:
 - produce ref assemblies even when there are errors outside method bodies (emitting error types when `EmitOptions.TolerateErrors` is set)
 
 ## Open questions
-- should explicit method implementations be included in ref assemblies?
+- should explicit method implementations be included in ref assemblies? (answer: no. The interfaces that are declared as implemented are what matter to consuming compilations)
 - Non-public attributes on public APIs (emit attribute based on accessibility rule)
 - ref assemblies and NoPia
-- `/refout` and `/addmodule`, should we disallow this combination?
+- `/refout` and `/addmodule`, should we disallow this combination? (answer: yes. This will not be supported in C# 7.1)
```

Comment:
Are these still open questions? If not, consider moving to another section in the document.